src/test/java/com/puppycrawl/tools/checkstyle/internal/ArchUnitCyclesCheckTest.java

```diff
@@ -30,7 +29,6 @@
 import com.tngtech.archunit.lang.ArchRule;
 import com.tngtech.archunit.library.freeze.FreezingArchRule;
 
-@Disabled("until https://github.com/checkstyle/checkstyle/issues/13544")
```

Comment:
We can now reference https://github.com/checkstyle/checkstyle/issues/14053 in disabled annotation, please do this for all.

Reply:
done